presto-docs/src/main/sphinx/release/release-314.rst

```diff
@@ -0,0 +1,71 @@
+===========
+Release 314
+===========
+
+General Changes
+---------------
+
+* Fix queued queries counter in UI. (:issue:`894`)
+* Fix incorrect results for ``BETWEEN`` with a ``NULL`` min or max value. (:issue:`877`)
+* Fix query history leak in coordinator. (:issue:`939` :issue:`944`)
+* Improve performance of :func:`json_parse` function. (:issue:`904`)
+* Visualize plan structure in ``EXPLAIN`` output. (:issue:`888`)
+* Add support for positional access to ``ROW`` fields via the subscript
+  operator. (:issue:`860`)
+
+CLI Changes
+-----------
+
+* Add JSON output format. (:issue:`878`)
+
+Hive Connector Changes
+----------------------
+
+* Fix listing tables and views from Hive 2.3+ Metastore on certain databases,
+  including Derby and Oracle. This fixes ``SHOW TABLES``, ``SHOW VIEWS`` and
+  reading from ``information_schema.tables`` table. (:issue:`833`)
+* Fix handling of Avro tables with ``avro.schema.url`` defined in Hive
+  ``SERDEPROPERTIES``. (:issue:`898`)
+* Fix regression that caused ORC bloom filters to be ignored. (:issue:`921`)
+* Add support for reading LZ4 and ZSTD compressed Parquet data. (:issue:`910`)
+* Add support for writing ZSTD compressed ORC data. (:issue:`910`)
+* Add support for configuring ZSTD and LZ4 as default compression methods via the
+  ``hive.compression-codec`` config option. (:issue:`910`)
+* Do not allow inserting into text format tables that have a header or footer. (:issue:`891`)
+* Add ``textfile_skip_header_line_count`` and ``textfile_skip_footer_line_count`` table properties
+  for text format tables that specify the number of header and footer lines. (:issue:`845`)
+* Add ``hive.max-splits-per-second`` configuration property to allow throttling
+  the split discovery rate, which can reduce load on the file system. (:issue:`534`)
+* Support overwriting unpartitioned tables for insert queries. (:issue:`924`)
+
+PostgreSQL Connector Changes
+----------------------------
+
+* Support PostgreSQL arrays declared using internal type
+  name, for example ``_int4`` (rather than ``int[]``). (:issue:`659`)
+
+Elasticsearch Connector Changes
+-------------------------------
+
+* Add support for mixed-case field names. (:issue:`887`)
+
+JDBC-Based Connectors Changes
+---------------------------------
```

Comment:
Header should match text length

Reply:
This is a library change, not a user visible connector change, so the title should be
```
Base-JDBC Connector Library Changes
-----------------------------------